d2bs/kolbot/libs/common/Pather.js

```diff
@@ -118,7 +118,7 @@ var Pather = {
 	recursion: true,
 
 	useTeleport: function () {
-		return this.teleport && !me.getState(139) && !me.getState(140) && !me.inTown && ((me.classid === 1 && me.getSkill(54, 1)) || me.getStat(97, 54));
+		return this.teleport && !Config.NoTele && !me.getState(139) && !me.getState(140) && !me.inTown && ((me.classid === 1 && me.getSkill(54, 1)) || me.getStat(97, 54));
```

Comment:
there is redundancy with Pather.teleport here.
i'd suggest removing this addition and just modify the block a few lines above to something like this:

```
var Pather = {
	teleport: !Config.NoTele,
```